service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NewHearingsAddedEventHandlerTest.java

```diff
@@ -0,0 +1,131 @@
+import uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences;
+import uk.gov.hmcts.reform.fpl.events.NewHearingsAddedEvent;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.event.EventData;
+import uk.gov.hmcts.reform.fpl.model.notify.hearing.NewNoticeOfHearingTemplate;
+import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
+import uk.gov.hmcts.reform.fpl.service.InboxLookupService;
+import uk.gov.hmcts.reform.fpl.service.config.LookupTestConfig;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.NewNoticeOfHearingEmailContentProvider;
+import uk.gov.hmcts.reform.fpl.service.representative.RepresentativeNotificationService;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+import uk.gov.hmcts.reform.fpl.utils.FixedTimeConfiguration;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.NOTICE_OF_NEW_HEARING;
+import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.CAFCASS_EMAIL_ADDRESS;
+import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.LOCAL_AUTHORITY_CODE;
+import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.LOCAL_AUTHORITY_EMAIL_ADDRESS;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createHearingBooking;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.callbackRequest;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {
+    NewHearingsAddedEventHandler.class,
+    JacksonAutoConfiguration.class,
+    LookupTestConfig.class,
+    FixedTimeConfiguration.class})
+class NewHearingsAddedEventHandlerTest {
+    private static final String CASE_REFERENCE = "12345";
+
+    @Autowired
+    private NewHearingsAddedEventHandler newHearingsAddedEventHandler;
+
+    @Autowired
+    private Time time;
+
+    @MockBean
+    private NotificationService notificationService;
+
+    @MockBean
+    private NewNoticeOfHearingEmailContentProvider newNoticeOfHearingEmailContentProvider;
+
+    @MockBean
+    private HearingBookingService hearingBookingService;
+
+    @MockBean
+    private InboxLookupService inboxLookupService;
+
+    @MockBean
+    private RepresentativeNotificationService representativeNotificationService;
+
+    private LocalDateTime futureDate;
+
+    @BeforeEach
+    void setUp() {
+        futureDate = time.now().plusDays(1);
+    }
+
+    @Test
+    void shouldSendNotificationToLACafcassAndRepresentativesWhenNewHearingIsAdded() {
+        final ObjectMapper mapper = new ObjectMapper();
```

Comment:
I would recommend using the same object mapper the code is using (via Injection), this can lead to differences how the code is serialised / deserialised.